config/settings.yaml.example

```diff
@@ -23,5 +23,14 @@
 :domain: 'localdomain.net'
 :fqdn: 'localhost.localdomain.net'
 
-# Should ANSI color codes be used in logs
-:colorize_logging: false
+# Loggers can be configured individually, they can be turned off completely by using key
+# 'enabled' or you can adjust its logging 'level'. If you specify a level higher than root
+# logger has, messages below this level are ignored. If you specify lower level than root
+# they are still ignored but you can add appender in Foreman::Logging just for this logger
```

Comment:
This doesn't seem like very useful advice for users... is there any way to make this work by default, or should we just recommend that they keep the root logger level below any other debug they want?